recipes/flit/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.9" %}
+
+package:
+  name: flit
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/takluyver/flit.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python >=3
```

Comment:
Don't use the ` >3`. That might be the cause of the missing PY version string. Instead add a `skip: True  # [py2k]` at the build section above.